website/content/docs/agent/autoauth/methods/approle.mdx

```diff
@@ -35,3 +35,62 @@ cached.
   containing the output of the secret ID retrieval endpoint for the role (e.g.
   `auth/approle/role/webservers/secret-id`) and the creation path for the
   response-wrapping token must match the value set here.
+
+## Example Configuration
+
+An example configuration, using approle to enable [auto-auth](/docs/agent/autoauth) and creating both a plaintext token sink and [encrypted token sink file](/docs/agent/autoauth#wrap_ttl), follows:
```

Comment:
I would avoid phrasing it as "encrypted token sink", since there is an encrypted DH option distinct from using response wrapping.

Reply:
Updated to response wrapped vs encrypted. thanks for the 👀 !